app/src/org/commcare/google/services/analytics/GoogleAnalyticsUtils.java

```diff
@@ -400,7 +400,7 @@ private static Tracker getTracker() {
     }
 
     private static boolean analyticsDisabled() {
-        return !CommCarePreferences.isAnalyticsEnabled();
+        return true;
```

Comment:
hm, just so this doesn't come back and bite us with a bad backmerge later: is there a way to just tag this by not using the core package id?